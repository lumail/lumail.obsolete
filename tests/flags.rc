set_selected_folder('output/folders/flags')
io.write(('Messages: %d\n'):format(count_messages()))
local idx = 0
while idx < count_messages() do
    jump_index_to(idx)
    local msg = current_message()
    local flags = msg:flags()
    io.write('got flags: '..flags..'\n')
    idx = idx + 1
end
