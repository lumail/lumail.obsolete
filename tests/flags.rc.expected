Messages: 3
got flags: S
got flags: 
got flags: N
Exit: 0
